README.md

```diff
@@ -88,19 +88,48 @@ helm list # should display two charts installed
 
 ## Verification
 
-Each release of the chart is signed using *provenance files*.
-You can verify the chart by following the instructions below, depending on the registry.
+Each release of the chart is signed using *provenance files*. You can verify the chart by following the instructions below:
```

Comment:
```suggestion
Starting with 37.0.0, chart releases are signed using *provenance files*. One can verify the chart by following the instructions below:
```

Reply:
Yes, we use to "impersonate" messages. In this case you can also say:
```suggestion
Starting with 37.0.0, chart releases are signed using *provenance files*. Those can be verified using the following the instructions below:
```